tests/unit/test_upgrade.py

```diff
@@ -3,19 +3,34 @@
 import unittest
 from unittest.mock import Mock, PropertyMock, patch
 
+import httpx
+import pytest
+from charms.mongodb.v0.upgrade_helpers import UnitState
+from lightkube import ApiError
+from ops import StartEvent
 from ops.model import ActiveStatus, Relation
 from ops.testing import ActionFailed, Harness
 from parameterized import parameterized
+from tenacity import Future, RetryError
 
 from charm import MongoDBCharm
 from config import Config
+from k8s_upgrade import DeployedWithoutTrust, KubernetesUpgrade
 
 from .helpers import patch_network_get
 
 
+@pytest.fixture(autouse=True)
```

Comment:
can this be moved to a fixture to be used by all unit tests?

Reply:
I'll check, should be doable

Reply:
Can't find a way to do it.